src/astro/src/layouts/index.astro

```diff
@@ -14,4 +15,7 @@ const {title} = Astro.props;
   <Nav />
   <Front /> 
 </body>
+<footer>
```

Comment:
Doubly wrapped footer element.

Reply:
I removed from the component so now only wrapped once.